components/kubeflow/pytorch-launcher/src/launch_pytorchjob.py

```diff
@@ -28,18 +29,6 @@ def get_current_namespace():
     return current_namespace
 
 
-# Patch PyTorchJob APIs to align with k8s usage
-class V1PyTorchJob(V1PyTorchJob_original):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.openapi_types = self.swagger_types
-
-
-class V1PyTorchJobSpec(V1PyTorchJobSpec_original):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.openapi_types = self.swagger_types
-
```

Comment:
Are these no longer needed? Will this change break existing usage?

Reply:
I moved to using KubeflowOrgV1PyTorchJob and KubeflowOrgV1PyTorchJobSpec  from kubeflow-training to be inline with the training operator, therefore these classes are not required. I am not sure exactly how to test if it will break existing usage, or if this is relevant - any advice on this would be appreciated. @HumairAK @rimolive could you please advise?

Reply:
I would ignore concerns around breaking compatibility, as this component has not been updated in a really long time (and was thus moved to contrib as a result).